app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
Improve "code" blocks, it should be better to have a background or something.

See in http://localhost:3001/docs/cloud_hosting/cli